doc/source/data/dataset.rst

```diff
@@ -2,48 +2,55 @@
 
 .. note::
 
-    Before you proceed, note that Ray Data is available as **beta** in Ray 1.8+.
+    Before you proceed, note that Ray Datasets is available as **beta** in Ray 1.8+.
     Please file feature requests and bug reports on GitHub Issues or join the discussion
     on the `Ray Slack <https://forms.gle/9TSdDYUgxYs8SA9e8>`__.
 
-==============================================
-Ray Data: Distributed Data Loading and Compute
-==============================================
+==================================================
```

Comment:
I'm wondering, should we also introduce "Ray Data" package more generally? Though, the current intro seems technically accurate. We can probably revisit this at a later time.

Reply:
Ah, I see this is in your future changes proposal.

Reply:
That's (1) in the PR description for changes that will be in a future PR, primarily focusing on small, quick changes in this one.